### DISCUSSION TITLE
Can I have a new release on Major version 1?

### DISCUSSION THREAD

**Discussion by ylin20220913** [pelletier/go-toml#759] [Questions] [answered]
Can any maintainers to help me  publish a new tag `1.9.5` with this bug been fixed:  https://github.com/pelletier/go-toml/pull/608 ? 

The tag 1.9.5  can also  be set as current `master` branch. That would also work for us. 

Thank you!

**Comment by pelletier** [comment, marked as answer]
I will tag it shortly. The binaries will probably have to wait until tomorrow.  EDIT: here's `1.9.5`: https://github.com/pelletier/go-toml/releases/tag/v1.9.5
